common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -44,6 +44,7 @@ class WhiskConfig(
     val dockerImageTag = this(WhiskConfig.dockerImageTag)
 
     val invokerContainerNetwork = this(WhiskConfig.invokerContainerNetwork)
+    val invokerContainerPolicy = if (this(WhiskConfig.invokerContainerPolicy) == "") None else Some(this(WhiskConfig.invokerContainerPolicy))
```

Comment:
We can probably get around by the default value of `""` and the `if..else` by using `Option.apply` like this:

```
val invokerContainerPolicy = Option(this(WhiskConfig.invokerContainerPolicy))
```

and setting the default value to `null` (gasp)